draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -377,6 +377,9 @@ reordering, priority, parallelism, and header compression, all of which
 are supported by HTTP/2 {{RFC7540}} or its
 successors.
 
+If the protocol does not use HTTP/2 or its succesors, it MUST
+use HTTP 1.1 {{RFC7230}}. This will possibly result in worse performance.
```

Comment:
I don't much see the point in this paragraph.

Better to concentrate on the performance advantages of h2, as the previous paragraph already does.

Reply:
If there is a SHOULD, there should be a "if not". We don't have to give the second sentence, but then we haven't really explained why not MUST.

Reply:
There is no real "if not" here.  It an "or else". 

You could firm up the bit that explains why h2 is good with a stronger implication of "or else".  You don't have to use h2, but if you don't, your performance will suffer.  

I guess that you could phrase that as an "if not", but saying "if not, then understand that your performance might not meet with expectations".